src/style/style.css

```diff
@@ -6968,6 +6973,130 @@ fieldset[disabled] .header-search-input {
 textarea.header-search-input {
   height: auto;
 }
+
+/* ======================== */
+/* MAIN LOCALE CONTAINER    */
+/* ======================== */
+
+.main-locale-container {
+  position: absolute;
+  right: 130px;
+  top: 22px;
+  /* margin: 0 0 0 0; */
+}
+@media (max-width: 768px) {
+  .container {
+    max-width: 100%;
+  }
+}
+@media (min-width: 768px) and (max-width: 992px) {
+  .main-locale-container {
+    right: 110px;
+  }
+}
+@media (max-width: 768px) {
+  .main-locale-container {
+    position: absolute;
```

Comment:
This doesn't appear to align correctly in mobile (the dropdown overlaps off screen)
![image](https://user-images.githubusercontent.com/39800563/43417348-bdbb977c-9408-11e8-9c8a-44729941337b.png)

Reply:
Switched to bootstrap dropdown which lays out all options vertically, so it should look good on mobile now.